stats/cloud/collector.go

```diff
@@ -527,16 +537,48 @@ func (c *Collector) pushMetrics() {
 	c.bufferSamples = nil
 	c.bufferMutex.Unlock()
 
+	count := len(buffer)
 	logrus.WithFields(logrus.Fields{
-		"samples": len(buffer),
+		"samples": count,
 	}).Debug("Pushing metrics to cloud")
+	start := time.Now()
+
+	numberOfPackages := len(buffer) % int(c.config.MaxMetricSamplesPerPackage.Int64)
```

Comment:
hmm this seems wrong? it should be something like `ceil ( len(buffer) / MaxMetricSamplesPerPackage )`, right?

Reply:
:facepalm:

Reply:
fixed in b9573e4